v2/src/components/appInfoForm/style.css

```diff
@@ -157,15 +157,33 @@
 }
 
 .app-info-form-field-input {
-  border: 1px solid rgb(72, 72, 72);
+  border: 1px solid rgb(69, 69, 69);
+  background-color: rgb(0, 0, 0);
+  color: rgb(255, 255, 255);
   box-shadow: 0px 0px 0px 0px transparent;
   outline: none;
   transition: border .15s, box-shadow .15s;
+  padding: 8px 20px;
+  font-family: Roboto;
 }
 
-.app-info-form-field-input:focus {
-  border: 1px solid rgb(255, 128, 0);
-  box-shadow: 0px 0px 0px 2px rgb(255, 191, 128);
+.app-info-form-field-input:focus {  
+  border-color: rgb(255, 153, 51);
+  box-shadow: 0px 0px 3px 1px rgb(255, 153, 51, 0.8);
+}
+
+.app-info-form-field-input::placeholder, .app-info-form-field-input:-ms-input-placeholder, .app-info-form-field-input::-ms-input-placeholder {
+  color: rgb(90, 90, 90);
+}
+
+input.app-info-form-field-input:focus::placeholder {
+  color: transparent;
+}
+.app-info-form-field-input:focus:-ms-input-placeholder{
+  color: transparent;
+}
+.app-info-form-field-input:focus::-ms-input-placeholder {
+  color: transparent;
```

Comment:
tried combining these 3 rules into one, but it doesn't work. Probably, the parser breaks because of the multiple `:` on the `:focus::placeholder` if these 3 rules merged into one

Reply:
👍🏼